GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
+
+    guard let dict = plist as? Dictionary<String, String> else {
+      throw StringsFileParserError.InvalidFormat
+    }
+
     var encoding: NSStringEncoding = NSUTF16StringEncoding
     let fileContent = try NSString(contentsOfFile: path, usedEncoding: &encoding)
+
+    func extractKey(line: String) -> String? {
+      let regex: NSRegularExpression = {
+        do {
+          return try NSRegularExpression(pattern: "^\"([^\"]+)\"[ \t]*=", options: [])
+        } catch {
+          fatalError("Error while building Regular Expression for strings file parsing")
+        }
+      }()
+
+      let range = NSRange(location: 0, length: (line as NSString).length)
+      if let match = regex.firstMatchInString(line, options: [], range: range) {
+        return (line as NSString).substringWithRange(match.rangeAtIndex(1))
+      }
+      return nil
+    }
+
     let lines = fileContent.componentsSeparatedByCharactersInSet(NSCharacterSet.newlineCharacterSet())
```

Comment:
This is not needed anymore, we don't need to treat the `.strings` file as a textual file now that we load it as a dictionary